synfig-studio/ui/synfig-catalog.xml

```diff
@@ -2,14 +2,18 @@
 <glade-catalog name="synfigstudio" depends="gtk+" version="0.2" domain="synfigstudio">
 
   <glade-widget-classes>
+    <glade-widget-class name="gtkmm__CustomObject_widget_gammapattern" title="Gamma Pattern" />
```

Comment:
please add the "since" attribute :)

Reply:
Adding to _widget_gammapattern and _widget_link...

Also, I notice _widget_vector is missing the since attribute. Intentional?

Reply:
It's because it was defined since the beginning :)

Reply:
Ideally, I'd implemented all existent widgets in catalog at once. But I had other plans/priorities to do… :/

Reply:
OK, resolved :)